snaps/build-and-release-snaps-on-new-release.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+
+KUBE_VERSION="${KUBE_VERSION:-$(curl -L https://dl.k8s.io/release/stable.txt)}"
+# LAST_RELEASE_FILE keeps the last release we did.
+LAST_RELEASE_FILE="/var/tmp/last_k8s_patch_release"
+
+
+function check_for_release {
+  trigger='no'
+  if [ -f $LAST_RELEASE_FILE ]
+  then
+    LAST_RELEASED=`cat $LAST_RELEASE_FILE`
+    if [ "$LAST_RELEASED" != "$KUBE_VERSION" ]
+    then
+      echo "New release ($KUBE_VERSION) detected."
+      echo "$KUBE_VERSION" > $LAST_RELEASE_FILE
+      trigger='yes'
```

Comment:
What if build-and-release-k8s-snaps.sh crashes? e.g. snap store is down. This script will have updated the version in $LAST_RELEASE_FILE without that version ever being pushed or released. So we could end up skipping over a minor release if we're unlucky.